api/src/Page/Image.php

```diff
@@ -133,10 +133,9 @@ function _action_image() {
             if ($n < sizeof($images)) {
                 $this->_browser_cache();
                 $ext = pathinfo($images[$n], PATHINFO_EXTENSION);
-                $file = $this->has_arg('f') ? $images[$n] : str_replace('.'.$ext, 't.'.$ext, $images[$n]);
-                if (file_exists($file)) {
+                if (file_exists($images[$n])) {
```

Comment:
Given that we are removing the use of the 'f' flag in data collection/robot action snapshot requests, can we get rid of the "/f/1" in lots of places?